exercises/concept/expenses/expenses.go

```diff
@@ -0,0 +1,46 @@
+package expenses
+
+import "time"
+
+// Record describes a record of expense event.
+type Record struct {
+	Date     time.Time
+	Amount   float64
+	Category string
+}
+
+// DatePeriod describes date period.
+type DatePeriod struct {
+	From time.Time
+	To   time.Time
+}
+
+// Filter creates a new records collection by applying the predicate function to
+// records and keeping the records for which the function returns true.
+func Filter(in []Record, f func(Record) bool) (out []Record) {
```

Comment:
```suggestion
func Filter(in []Record, f func(Record) bool) []Record {
```

As much as possible, we try to discourage the use of named returns.